src/ApplicationInsights.ServiceFabric.Native/Net45/ReliableServiceCorrelatingServiceRemotingClientFactory.cs

```diff
@@ -0,0 +1,38 @@
+﻿namespace Microsoft.ApplicationInsights.ServiceFabric.Remoting.Activities
+{
+    using Microsoft.ServiceFabric.Services.Client;
+    using Microsoft.ServiceFabric.Services.Communication.Client;
+    using Microsoft.ServiceFabric.Services.Remoting;
+    using Microsoft.ServiceFabric.Services.Remoting.FabricTransport;
+    using Microsoft.ServiceFabric.Services.Remoting.FabricTransport.Client;
+    using System.Collections.Generic;
+
+    /// <summary>
+    /// Remoting client factory for constructing clients that can call actor services. Clients created by this factory pass correlation ids and relevant information
+    /// to the callee so diagnostic traces can be tagged with the relevant ids. This factory wraps and use <see cref="FabricTransportServiceRemotingClientFactory"/> for most of the
+    /// underying functionality. <see cref="ReliableServiceCorrelatingServiceRemotingClientFactory"/> calls <see cref="FabricTransportServiceRemotingClientFactory"/> to create an inner client, which
+    /// handles the main call transport and will be wrapped by a <see cref="CorrelatingServiceRemotingClient"/> object.
+    /// </summary>
+    public class ReliableServiceCorrelatingServiceRemotingClientFactory : CorrelatingServiceRemotingClientFactory
```

Comment:
I mean why not remove `abstract` modifier from `CorrelatingServiceRemotingClientFactory` and use it directly instead of `ReliableServiceCorrelatingServiceRemotingClientFactory `